README.adoc

```diff
@@ -172,10 +276,12 @@ If set to `true` and given a collection is not set in the map, an empty collecti
 Special field which can be used to determinate equality of 2 instances. When it is not defined the String representation of the 2 objects is used for `equals`
 - enumMappingMethod (default `name`)
 When an enum type is defined as a field, we need to define what type of representation is coded in the map.
+- mapNullToOptionalAbsent (default `false`)
+When it is true, when the value not set or absent in the given map it will return Optional.isPresent() false.
```

Comment:
"When it is true, when the value not set or absent in the given map it will return Optional.isPresent() false." This sentence should be refined. I have no idea what we mean by it. :)

Reply:
Maybe "When set to true and a value is not present, the proxy will return with an `Optional` value where the `.isPresent()` check will result in `false`."